src/app/components/AdaptiveTrimmer/AdaptiveDynamicTrimmer.tsx

```diff
@@ -35,6 +52,8 @@ export const AdaptiveDynamicTrimmer: FC<AdaptiveDynamicTrimmerProps> = ({
   getShortenedContent,
   extraTooltip,
   tooltipOverride,
+  debugMode = false,
+  minLength = 2,
```

Comment:
trimLongString should validate inputs or handle negative numbers better, even tho this avoids the edge case

Reply:
Added that.